fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -0,0 +1,72 @@
+module FastlaneCore
+  class ToolCollector
+    TOOLS = %w(fastlane fastlane_core deliver snapshot frameit pem sigh produce cert gym pilot credentials_manager spaceship scan supply watchbuild match screengrab)
+    HOST_URL = "https://fastlane-enhancer.herokuapp.com/"
+
+    attr_reader :error
+
+    def did_launch_action(name)
```

Comment:
The plan is to change `fastlane`'s implementation to inherit from this class in a follow-up PR. I agree that the name could be better, but I think we may need to keep it as-is because of that.